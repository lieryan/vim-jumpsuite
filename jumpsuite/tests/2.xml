<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
	<testsuite errors="0" failures="15" name="comments.tests.test_api.APITests-20190208123140" skipped="0" tests="18" time="2.180" timestamp="2019-02-08T12:31:42">
		<testcase classname="comments.tests.test_api.APITests" name="test_form_0_do_something" time="0.103" timestamp="2019-02-08T12:31:40">
			<failure message="302 != 200" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/Users/lieryan/.virtualenvs/myenv/lib/python3.6/site-packages/parameterized/parameterized.py", line 392, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/home/user/Projects/myapp/comments/tests/test_api.py", line 58, in test_form
    self.assertEqual(302, response.status_code)
AssertionError: 302 != 200
]]>			</failure>
		</testcase>
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testsuite>
</testsuites>
