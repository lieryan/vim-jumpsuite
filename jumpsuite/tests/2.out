FAILED. Ran 18 tests in 2.180s (errors=0, failures=15, skipped=0)
/home/user/Projects/myapp/comments/tests/test_api.py:58: test_form : AssertionError: 302 != 200 : self.assertEqual(302, response.status_code)
